doc/install/ceph-deploy/quick-ceph-deploy.rst

```diff
@@ -289,12 +289,16 @@ example::
 
 .. topic:: Exercise: Locate an Object
 
-   As an exercise, lets create an object. Specify an object name, a path to
+   As an exercise, let's create an object. Specify an object name, a path to
    a test file containing some object data and a pool name using the
-   ``rados put`` command on the command line. For example::
+   ``rados put`` command on the command line. Let's also enable placement group
+   autoscaling so that the cluster will automatically set an appropriate number
+   of placement groups for the pool. For example::
 
      echo {Test-data} > testfile.txt
-     ceph osd pool create mytest
+     ceph mgr module enable pg_autoscaler
```

Comment:
`pg_autoscaler` is enabled by default in octopus. probably we can dispense with this command?

Reply:
That makes sense (and having it enabled by default will be newbie friendlier, too, hopefully). Have now removed the autoscaling references in a new commit.